internal/dcache/rpc/server/handler.go

```diff
@@ -46,44 +56,743 @@ var _ service.ChunkService = &ChunkServiceHandler{}
 
 // ChunkServiceHandler struct implements the ChunkService interface
 type ChunkServiceHandler struct {
-	locks *common.LockMap
-	// more fields can be added here as needed
+	locks   *common.LockMap
+	fsIDMap map[string]*rvInfo // map to store the fsID to rvInfo mapping
+}
+
+type rvInfo struct {
+	rvID     string       // rv0, rv1, etc.
+	cacheDir string       // cache dir path for this RV
+	mvMap    sync.Map     // sync map of MV id against mvInfo
+	mvCount  atomic.Int64 // count of MVs for this RV, this should be updated whenever a MV is added or removed from the sync map
+}
+
+type mvInfo struct {
```

Comment:
mvInfo must have the mvName as the first field, followed by all the component RVs.
I think it will be useful to store two values

localRv    string
remoteRVs []string

so that we know what is the local and remote RVs for this MV.
Let me look at the rest of code to see what makes better sense.

Reply:
Added mvName